common/common_test.go

```diff
@@ -34,6 +34,10 @@ func TestIsValidCryptoAddress(t *testing.T) {
 	if err != nil && !b {
 		t.Errorf("Common IsValidCryptoAddress error: %s", err)
 	}
+	b, err = IsValidCryptoAddress("bc1qw508d6qejxtdg4y5r3zarvaly0c5xw7kv8f3t4", "bTC")
```

Comment:
I think bech32 addresses also introduced zeroes so your regex needs to become:
`^(bc1|[13])[a-zA-HJ-NP-Z0-9]{25,39}$`

Your test is invalid because `b` is false as it contains zeroes.

[Source 1](https://en.bitcoin.it/wiki/Bech32)
[Source 2](https://regexland.com/regex-bitcoin-addresses/)
[Source 3](https://ihateregex.io/expr/bitcoin-address/)